__tests__/app.test.js

```diff
@@ -75,7 +75,52 @@ describe('GET/api/articles/:article_id', () => {
     .get('/api/articles/90')
     .expect(404)
     .then(({body}) => {
-      expect(body).toHaveProperty('msg', 'no article found with the given id')
+      expect(body).toHaveProperty('msg', 'article does not exist')
     })
   })
-})
\ No newline at end of file
+})
+
+describe('GET/api/articles/:article_id/comments', () => {
+  test('200: responds with an array of comment objects relating to the given article_id sorted from newest to oldest', () => {
+    return request(app)
+    .get('/api/articles/9/comments')
+    .expect(200)
+    .then(({body}) => {
+      expect(body.comments.length).toBe(2)
+      expect(body.comments).toBeSortedBy('created_at', {descending: true})
+      body.comments.forEach(comment => {
+        expect(comment).toHaveProperty('comment_id', expect.any(Number))
+        expect(comment).toHaveProperty('votes', expect.any(Number))
+        expect(comment).toHaveProperty('created_at', expect.any(String))
+        expect(comment).toHaveProperty('author', expect.any(String))
+        expect(comment).toHaveProperty('body', expect.any(String))
+        expect(comment).toHaveProperty('article_id', expect.any(Number))
```

Comment:
since you're requesting comments on the article with id 9, you can assert this property to be 9 exactly!